tests/protocol/test_albumart.py

```diff
@@ -0,0 +1,53 @@
+from io import BytesIO
+from mopidy_mpd.protocol import album_art
+from unittest import mock
+from mopidy.models import Album, Track, Image
+
+from tests import protocol
+
+
+def mock_get_images(self, uris):
+    result = {}
+    for uri in uris:
+        result[uri] = [Image(uri="dummy:/albumart.jpg", width=128, height=128)]
+    return result
+
+
+class AlbumArtTest(protocol.BaseTestCase):
```

Comment:
Needs tests for the image width sorting, caching, and `offset`. And all the many ways urlopen can fail.